alphagov/paas-product-pages
# Maintenance updates

- update dependencies
- update browserlist db
- fix React ' keys' warning

## How to test

- checkout branch
- run `npm install`

To test local development
- run `npm run local`
- check pages

To test production build
- run `npm run build`
- `cd` to `out` directory
- run `npx serve`
- check pages
